code/modules/antagonists/revolution/revolution.dm

```diff
@@ -303,6 +303,18 @@
 		new_target.target = M
 		new_target.update_explanation_text()
 		objectives += new_target
+
+	var/untracked_protagonists = SSstation.protagonists
+	for(var/datum/objective/mutiny/O in objectives)
```

Comment:
better var names, please

https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names